From a63772f7fff4f394ee2770eeb139f94083bc02cc Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Fri, 13 Oct 2017 19:42:24 +0300
Subject: [PATCH 177/448] staging: fsl-dpaa2/eth: Function rename

Rename dpaa2_eth_xdp_xmit() to dpaa2_eth_xdp_tx() to avoid
possible name conflicts with the (future) implementation of
ndo_xdp_xmit.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
index bf0d578..f77194a 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -221,10 +221,10 @@ static void free_rx_fd(struct dpaa2_eth_priv *priv,
 	return skb;
 }
 
-static int dpaa2_eth_xdp_xmit(struct dpaa2_eth_priv *priv,
-			      const struct dpaa2_fd *fd,
-			      void *buf_start,
-			      u16 queue_id)
+static int dpaa2_eth_xdp_tx(struct dpaa2_eth_priv *priv,
+			    const struct dpaa2_fd *fd,
+			    void *buf_start,
+			    u16 queue_id)
 {
 	struct dpaa2_eth_fq *fq;
 	struct rtnl_link_stats64 *percpu_stats;
@@ -366,8 +366,8 @@ static void dpaa2_eth_rx(struct dpaa2_eth_priv *priv,
 				release_fd_buf(priv, ch, addr);
 				goto drop_cnt;
 			case XDP_TX:
-				if (dpaa2_eth_xdp_xmit(priv, fd, vaddr,
-						       queue_id)) {
+				if (dpaa2_eth_xdp_tx(priv, fd, vaddr,
+						     queue_id)) {
 					dma_unmap_single(dev, addr,
 							 DPAA2_ETH_RX_BUF_SIZE,
 							 DMA_BIDIRECTIONAL);
-- 
1.7.1

